clojure-mode.el

```diff
@@ -1009,7 +1009,7 @@ nil."
                     (one-or-more (any whitespace "\n")))
       ;; why is this here? oh (in-ns 'foo) or (ns+ :user)
       (zero-or-one (any ":'"))
-      (group (one-or-more (not (any "()\"" whitespace))) word-end)))
+      (group (one-or-more (not (any "()\"" whitespace))) symbol-end)))
```

Comment:
Our symbol definition is inherited from elisp so it's at least better than using word characters.  I can't think of any notable differences in clojure's and elisps notion of what a symbol is.